dependency-manager-tests/spm/SPMProject/ViewController.swift

```diff
@@ -18,6 +19,7 @@ internal class ViewController: UIViewController {
             trackingConsent: .granted,
             configuration: Datadog.Configuration
                 .builderUsing(clientToken: "abc", environment: "tests")
+                .enableCrashReporting(using: DDCrashReportingPlugin())
```

Comment:
shouldn't this be enable by default ?

Reply:
is it so in Android?
as a principle, nothing is enabled by default.